.github/workflows/release.yml

```diff
@@ -82,7 +82,7 @@ jobs:
       - id: base
         name: Base
         run: |
-          echo build_version=$(test ${{ github.ref_name }} == "v2" && echo "v2" || echo "v1") >> $GITHUB_OUTPUT
+          echo build_version=$(test ${{ github.ref_name }} == "main" && echo "v2" || echo "v1") >> $GITHUB_OUTPUT
```

Comment:
I assume we left this, so that we can re-use the build matrix for a v3 release in the future?

Reply:
I left this in case we need to do a v1 release due to sec issues